src/main/java/net/earthcomputer/clientcommands/command/ListenCommand.java

```diff
@@ -0,0 +1,269 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.google.gson.stream.JsonWriter;
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.Message;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.SimpleCommandExceptionType;
+import it.unimi.dsi.fastutil.objects.ReferenceOpenHashSet;
+import it.unimi.dsi.fastutil.objects.ReferenceSet;
+import net.earthcomputer.clientcommands.*;
+import net.earthcomputer.clientcommands.features.MappingsHelper;
+import net.earthcomputer.clientcommands.features.PacketDumper;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.ChatFormatting;
+import net.minecraft.core.Holder;
+import net.minecraft.core.Registry;
+import net.minecraft.network.chat.ClickEvent;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.HoverEvent;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.protocol.Packet;
+import net.minecraft.network.protocol.PacketFlow;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.ChunkPos;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.lang.invoke.VarHandle;
+import java.lang.reflect.Array;
+import java.lang.reflect.InaccessibleObjectException;
+import java.lang.reflect.Modifier;
+import java.time.Instant;
+import java.util.*;
```

Comment:
Wildcard import again, you should change your wildcard import limit in intellij to something large like 99999